src/article.js

```diff
@@ -27,6 +20,7 @@ class Article extends Component {
           </a>
           <h6>{this.props.category}</h6>
           {/* check date sorted desc <h6>{this.props.date}</h6> */}
+          {/* would have been a nice bonus to render the date */}
```

Comment:
Looks like you rendered the date to check that sorting worked - but you don't actually add it to the prod UI 😢